Update New-DlpComplianceRule.md (#9223)
MicrosoftDocs/office-docs-powershell: exchange/exchange-ps/exchange/New-DlpComplianceRule.md

```

### -ExceptIfFromMemberOf

> The ExceptIfFromMemberOf parameter specifies an exception for the DLP rule that looks for senders' email in the lists given as values to this param. You can give multiple lists to this parameter.

Reviewer: ```suggestion
The ExceptIfFromMemberOf parameter specifies an exception for the DLP rule that looks for senders' emails in the lists given as values to this parameter. You can give multiple lists to this parameter.
```

```

### -FromMemberOf

> The FromMemberOf parameter specifies an condition for the DLP rule that looks for senders' email in the lists given as values to this param. You can give multiple lists to this parameter.

Reviewer: ```suggestion
The FromMemberOf parameter specifies a condition for the DLP rule that looks for senders' emails in the lists given as values to this parameter. You can give multiple lists to this parameter.
```
